src/Pages/Home/Home.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import WelcomeMessage from './WelcomeMessage.js';
 
 const Home = () => {
 
     return (
         <div>
-            <p>Home Page</p>
+            <WelcomeMessage></WelcomeMessage>
```

Comment:
Can you change this line to `<WelcomeMessage />` since it is a component with no children or content inside